src/redisjson.rs

```diff
@@ -509,6 +532,7 @@ pub mod type_methods {
     pub unsafe extern "C" fn rdb_save(rdb: *mut raw::RedisModuleIO, value: *mut c_void) {
         let json = &*(value as *mut RedisJSON);
         raw::save_string(rdb, &json.data.to_string());
+        raw::save_unsigned(rdb, 0);
```

Comment:
@rafie @oshadmi why do you store this fake 0?

Reply:
I will remove it (I think it was before we had db ver `3`)